lib/location.js

```diff
@@ -97,6 +97,7 @@ var LocationState = State.extend(function (self, name, opts) {
         characters_per_page:160,
         next_text:"Next",
         previous_text:"Previous",
+        skip_text: "Skip",
```

Comment:
We'd set `skip_text` to not null when we instantiate the location state (e.g. `new LocationState("my_state", {skip: "Skip"})`).